.github/workflows/shared.yml

```diff
@@ -0,0 +1,45 @@
+name: EdFi.Admin.DataAccess Shared workflow
+
+on:
+  workflow_call:
+    inputs:
+      publish:
+        required: true
+        type: boolean
+
+
+env:
+  INFORMATIONAL_VERSION: "5.4"
+  CONFIGURATION: "Release"
+  AZURE_ARTIFACT_URL: "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
+  AZURE_ARTIFACT_NUGET_KEY: ${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }}
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
@bptillman could you please use commit hash instead of version number https://techdocs.ed-fi.org/display/EDFIODS/Guidelines+for+Use+of+GitHub+Actions